src/Altinn.Profile.Integrations/SblBridge/User.Favorites/InternalServerErrorException.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Altinn.Profile.Integrations.SblBridge.User.Favorites
+{
+    /// <summary>
+    /// Represents an exception that occurs when an internal server error is encountered.
+    /// </summary>
+    [Serializable]
```

Comment:
I believe this attribute is deprecated, right? Or no longer needed or something. We generally no longer serialize exceptions with a serializer. There are specialized mechanisms.